extensions/notebook/package.json

```diff
@@ -251,36 +251,41 @@
 			],
 			"standardKernels": [
 				{
-					"name": "Python 3",
-					"connectionProviderIds": []
-				},
-				{
-					"name": "PySpark",
+					"name": "pyspark3kernel",
+					"displayName": "PySpark3",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "PySpark3",
+					"name": "pysparkkernel",
+					"displayName": "PySpark",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "Spark | R",
+					"name": "sparkkernel",
+					"displayName": "Spark | Scala",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
 				},
 				{
-					"name": "Spark | Scala",
+					"name": "sparkrkernel",
+					"displayName": "Spark | R",
 					"connectionProviderIds": [
 						"HADOOP_KNOX",
 						"MSSQL"
 					]
+				},
+				{
+					"name": "python3",
+					"displayName": "Python 3",
```

Comment:
Can this be different on other platforms? Are there cases where we rely on this / are we resilient to this changing?

Reply:
cases doesn't matter. we will always convert to lowercase to compare